pom.xml

```diff
@@ -153,6 +153,7 @@
         <tini.version>0.9.0</tini.version>
         <marathon.client.version>0.4.9</marathon.client.version>
         <skipLicenseHeadersCheck>false</skipLicenseHeadersCheck>
+        <rabbitmq.image.version>rabbitmq:3.6.9-management</rabbitmq.image.version>
```

Comment:
Dermot wants this set to latest three part release version such as rabbitmq:3.6.9-management